src/font/Atlas.zig

```diff
@@ -291,18 +291,17 @@ pub fn grow(self: *Atlas, alloc: Allocator, size_new: u32) Allocator.Error!void
     const data_old = self.data;
     const size_old = self.size;
 
-    // Allocate our new data
-    self.data = try alloc.alloc(u8, size_new * size_new * self.format.depth());
+    try self.nodes.ensureUnusedCapacity(1);
```

Comment:
You forgot to pass the allocator here-- test fails because this code can't even compile.